### ISSUE TITLE
News index page has several images with missing alternative text

### ISSUE THREAD

**Issue by axamei** [pulibrary/pul_library_drupal#1380]
An audit of /news produced several errors related to missing alt text (Img element is the only content of the link, but is missing alt text. The alt text should describe the purpose of the link.)

List of links to news items:
```<a href="/file/ps-library-editionjpg" class="library-news-link">
<a href="/news/2014-12-15/imperial-portraits-marquand-library" class="library-news-link">
<a href="/news/2014-11-20/prepare-your-case-law-school-preview" class="library-news-link">
<a href="/news/2014-11-11/princetons-government-depository-turns-130" class="library-news-link">
<a href="/news/2014-10-22/mapping-tibet" class="library-news-link">
<a href="/news/2014-10-03/borrowdirect-expands-through-borrowdirect-plus" class="library-news-link">
<a href="/news/2014-09-03/university-archives-launches-digital-repository-senior-theses" class="library-news-link">
<a href="/news/2014-09-03/hokusai-manga-marquand-library" class="library-news-link">
<a href="/news/2014-08-18/nova-caesarea-cartographic-record-garden-state-1666-1888" class="library-news-link">
<a href="/news/2014-06-12/johns-hopkins-joins-borrow-direct" class="library-news-link">
<a href="/news/2014-05-29/building-books-architecture-children" class="library-news-link">
<a href="/news/2014-04-14/research-data-management-services-pul" class="library-news-link">
<a href="/news/2014-02-26/latin-american-medals-orders-tour-today-may-29" class="library-news-link">
<a href="/news/2013-12-15/recap-has-new-executive-director" class="library-news-link">
<a href="/news/2013-12-10/new-history-graduate-library-study-opens" class="library-news-link">
<a href="/news/2013-10-30/sharing-research-data-when-what-form-whom-and-what-cost" class="library-news-link">
<a href="/news/2013-10-21/woody-allen-screenwriter-work" class="library-news-link">
<a href="/news/2013-10-10/class-2013-senior-theses-now-available-dataspace" class="library-news-link">
<a href="/news/2013-10-07/remediating-avant-garde-magazines-and-digital-archives-princeton-university-library" class="library-news-link">
<a href="/news/2013-09-23/egyptian-film-posters-debut-new-library-exhibit" class="library-news-link">
<a href="/news/2013-09-19/little-magazines-make-big-splash-architecture-library" class="library-news-link">
<a href="/news/2013-09-18/mendel-music-library-celebrates-stravinskys-rite-spring-100" class="library-news-link">
<a href="/news/2013-09-16/chat-live-librarian-today" class="library-news-link">
<a href="/news/2013-09-09/pul-welcomes-incoming-grads" class="library-news-link">
<a href="/news/2013-06-05/pul-welcomes-new-slavic-studies-librarian" class="library-news-link">
<a href="/news/2013-06-03/engineering-library-hosts-class-exhibition-german-thin-shells" class="library-news-link">
<a href="/news/2013-05-28/alumnus-henry-wendt-and-wife-holly-donate-historic-world-map-collection" class="library-news-link">
<a href="/news/2013-05-06/pul-archive-team-wins-finding-aid-award" class="library-news-link">
<a href="/news/2013-09-04/annotated-books-digitized" class="library-news-link">
<a href="/news/2013-03-25/princyclopedia-2013-journey-centre-earth" class="library-news-link">
<a href="/news/2013-01-22/highlighted-database-smithsonian-global-sound" class="library-news-link">
<a href="/news/2013-01-02/library-acquires-18th-century-handel-score" class="library-news-link">
<a href="/news/2012-12-18/mudd-library-awarded-grant-digitize-records-cold-war" class="library-news-link">
<a href="/news/2012-12-18/new-map-and-geospatial-information-center" class="library-news-link">
<a href="/news/2012-09-18/fazlur-kahn-%E2%80%93-structural-artist-urban-building-form" class="library-news-link">
<a href="/news/2012-09-18/firestone-bookfinders-service" class="library-news-link">
<a href="/news/2012-09-18/first-x-then-y-now-z-landmark-thematic-maps" class="library-news-link">
<a href="/news/2012-07-27/check-out-library-ipad" class="library-news-link">```
